saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap

```diff
@@ -37876,7 +37876,7 @@ exports[`Storyshots Views / Products / Product list default 1`] = `
             <td
               class="MuiTableCell-root-id MuiTableCell-body-id Component-textRight-id"
             >
-              XAG 274.99
+              XAG274.99
```

Comment:
Hmm, that's unexpected.

Reply:
I just did `npm test -- -u` as @dominik-zeglen suggested.

Reply:
@dominik-zeglen Can you look into this?

Reply:
Old node version